README.adoc

```diff
@@ -31,7 +31,7 @@ You will learn how to document and filter RESTful APIs from annotations, POJOs,
 files by using MicroProfile OpenAPI.
 
 The OpenAPI specification, previously known as the Swagger specification, defines a standard interface
-for exposing RESTful APIs. This specification gives both humans and computers a better understanding
+for documenting and exposing RESTful APIs. This specification gives both humans and computers a better understanding
 of the functionalities of services without requiring direct access to source code or documentation.
```

Comment:
Reading this last sentence over again, it seems kind of strange to say that computers could understand something, right? I've suggested an edit below, but feel free to take it or leave it. The text as it stands is grammatically correct.

This specification defines the functionalities of services for both humans and computers without requiring direct access to source code or documentation.

Reply:
Btw, if this is a quote, we should handle it as such.  If not, we should rephrase.

Essentially, the specification helps APIs to be documented in a well-defined format that allows humans and machines to understand or "process" them in an easier manner.

Reply:
Maybe..."This specification **enables** both humans and computers **to process** the functionalities of services without requiring direct access to source code or documentation."?

If not, though, we can leave it how it is. :)

Reply:
I guess we've just gotten used to personifying inanimate objects. I changed it to: "This specification allows both humans and computers to understand or process the functionalities of services without requiring direct access to underlying source code or documentation" based on both of your suggestions. :)